gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+rm -f *.so
+
+CURR_DIR=`pwd`
+cd $CURR_DIR
+
+script_real_path=`realpath $0`
+script_home=`dirname $script_real_path`
+
+# generate and fetch ./com_gemstone_gemfire_internal_NanoTimer.h
+cd $script_home/../../../../../../../../../../gemfire-core/src/main/java/
+javah  com.gemstone.gemfire.internal.NanoTimer
+mv ./com_gemstone_gemfire_internal_NanoTimer.h $script_home/
+cd -
+if [ -e $script_home/com_gemstone_gemfire_internal_NanoTimer.h ];then
+  echo "SUCCESS: com_gemstone_gemfire_internal_NanoTimer.h generated "
+fi
+
+# generate and fetch com_pivotal_gemfirexd_internal_GemFireXDVersion.h
+cd $script_home/../../../../../
```

Comment:
Just for better readability - I think we should define some variables at the top using the realpath of this script, as in relative to this, something like $script_home for all and 'cd' to those path. Instead of ../../../ ... etc ...

Reply:
fixed in new commit